elixir/apps/domain/lib/domain/config.ex

```diff
@@ -115,16 +115,25 @@ defmodule Domain.Config do
 
   ## Feature flag helpers
 
+  defp feature_enabled?(feature) do
+    compile_config!(Definitions, feature) ||
+      compile_config!(Definitions, :all_features_enabled, %{})
```

Comment:
I think it's better not to add global flag and have a better control on individual features, it's just an additional configuration logic that we won't need (always can add more env vars in terraform)

Reply:
Ok, I figured we wanted all the features enabled in our staging/QA/whatever environments and it would be easier to put one ENV var in our scripts than X number of them.  But I can get rid of it.

Reply:
Actually, won't this make it a pain for the local development environments if we have to set individual feature flags?

Reply:
I did not notice in the last PR but we need to add application configuration
```
# config/config.ex
config :domain, :enabled_features,
  traffic_filters: true,
  ...

# config/runtime.exs
config :domain, :enabled_features,
  traffic_filters: compile_config!(Definitions, :feature_traffic_filters_enabled),
  ...
```

This will ensure that everything is enabled in dev/test mode (and you can disable it with `Domain.Config.put_env_override/3`) but then enabled individually when deployed via terraform / in `docker-compose.yaml`.

The reason to do that is that `compile_config!/2` is a bit costly, it reads from all the places (from app env to system env) so we can resolve all of the flags once at boot time and then just read them.